### PR TITLE
Fix/4565 shapefiles germany

### LINKED ISSUE #4565: Map in statistics is not filled with data
### Bug Description

In statistics you have the option so get the data also shown on the map. Unfortunately now data were shown on the map.

### Steps to Reproduce
1. Go to statistics
2. Select the needed data and as visualization the map
3. the map stays empty

### Expected Behavior

All datas are shown on the map or the option for a map is hidden if its not possible to show the selected data.

### Screenshots

grafik

### System Details
* Device: Win10
* SORMAS version: 1.56.1
* Android version/Browser: FF

### PR DESCRIPTION
**Pull request by JonasCir**
closes #4565
This PR:

- introduces shapefiles for Germany
- provides robust logic to handle the various coordinate systems possible for shapefiles and converts them to GPS lat/lon
- removes duplicate logic in region and district file loading by moving them into a new service

### CODE REVIEW

**Comment by DavidBaldsiefen** [comment]
Can you elaborate how to test this? What region/district/community can I use, or is only the persons address relevant?

**Comment by JonasCir** [comment]
You need the infrastructure data for Germany, I send you the files

**Comment by JonasCir** [comment]
```
devmode=true
survnet.url=127.0.0.1
# SORMAS-Docker Deutschland:
country.locale=de-DE
#country.locale=en-US
csv.separator=;
country.epidprefix=DEU
country.name=germany
geocodingServiceUrlTemplate=https://sg.geodatenzentrum.de/gdz_geokodierung_bund__11a1c01f-cef9-3b53-2ba3-94c81caab5c3/geosearch.json?query=$%7Bstreet%7D+$%7BhouseNumber%7D,$%7BpostalCode%7D+$%7Bcity%7D&filter=typ:haus&count1
geocodingLongitudeJsonPath=$.features[0].geometry.coordinates[0]
geocodingLatitudeJsonPath=$.features[0].geometry.coordinates[1]
createDefaultEntities=true
```

File: sormas-backend/src/main/java/de/symeda/sormas/backend/region/GeoShapeService.java [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+import javax.ejb.LocalBean;
+import javax.ejb.Stateless;
+import java.io.IOException;
+import java.net.URL;
+import java.util.*;
+
+@Stateless
+@LocalBean
+public class GeoShapeService {
```
Comment (MartinWahnschaffe) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/region/GeoShapeService.java:45]: Name Helper. Don't treat as bean

**Review by MartinWahnschaffe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**